spec/features/inputs/nested_select_input_spec.rb

```diff
@@ -193,12 +193,12 @@
 
       register_form(Invoice, false) do |f|
         f.input :city_id, as: :nested_select,
-                          level_1: { attribute: :country_id },
-                          level_2: {
-                            attribute: :region_id,
-                            filters: { name_contains: 'Met' }
-                            },
-                          level_3: { attribute: :city_id }
+                       level_1: { attribute: :country_id },
+                       level_2: {
+                         attribute: :region_id,
+                         filters: { name_cont: 'Met' }
+                       },
+                       level_3: { attribute: :city_id }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Align the elements of a hash literal if they span more than one line.